docs/element-templates/README.md

```diff
@@ -347,6 +368,8 @@ Custom Fields may have a number of constraints associated with them:
 * `maxLength`: Maximal length for the input
 * `pattern`: Regular expression to match the input against
 
+Note that the `constraints` configuration option only works if a custom `type` is configured and therefore a custom field will be rendered.
```

Comment:
I got what you want to express, but `custom type` sounds a bit weird to me, since you can still only specify the supported types (String, Text, ...), no further, "custom" one.

Maybe

> Note that the `constraints` configuration option only works if a `type` is configured in the template JSON and therefore a custom field will be rendered.

?